spec/koans.js

```diff
@@ -407,12 +410,12 @@ describe('destructuring function parameters. ', () => {
 
```

Comment:
class LexicallyBound {

    getFunction() {
      return () => this;
    }

    getArgumentsFunction() {
      return () => this.arguments;
    }
  }

Reply:
let expected = fn.call(anotherObj);